src/api/search/jest.setup.js

```diff
@@ -1 +1 @@
-process.env = { ...process.env, MOCK_ELASTIC: '1' };
+process.env = { ...process.env, MOCK_ELASTIC: '1', LOG_LEVEL: 'error' };
```

Comment:
I didn't add the `LOG_LEVEL` in my PR, but commented that anyone could do this locally. 
I guess its up for discussion if we wanted to keep this or not.

Reply:
I appreciated the `LOG_LEVEL` hint in your PR. My console was flooded with stuff until I changed that variable

Reply:
What I usually do is add a `console.log()` to say, `we have our logs on silent, but you can change in file x/y/z to error or debug if you want`, and then make this `silent`, or it spams too much.

Reply:
Yea I think that is a good idea. Even in `error` mode. It still print a whole bunch of stuff.